Mobile Buy SDK/Mobile Buy SDK/Additions/NSDateFormatter+BUYAdditions.m

```diff
@@ -38,8 +38,29 @@ + (NSDateFormatter*)dateFormatterForShippingRates
 + (NSDateFormatter*)dateFormatterForPublications
 {
 	NSDateFormatter *dateFormatter = [[NSDateFormatter alloc] init];
+	
+	if (![self is24hTimeFormat]) {
+		[dateFormatter setLocale:[NSLocale localeWithLocaleIdentifier:@"en_GB"]];
+	}
+	
 	dateFormatter.dateFormat = @"yyyy-MM-dd'T'HH:mm:ssZ";
 	return dateFormatter;
 }
 
++ (BOOL)is24hTimeFormat
+{
+	NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
+	[formatter setLocale:[NSLocale currentLocale]];
+	[formatter setDateStyle:NSDateFormatterNoStyle];
+	[formatter setTimeStyle:NSDateFormatterShortStyle];
+	
+	NSString *dateString = [formatter stringFromDate:[NSDate date]];
+	
+	NSRange amRange = [dateString rangeOfString:[formatter AMSymbol]];
+	NSRange pmRange = [dateString rangeOfString:[formatter PMSymbol]];
+	BOOL is24h = (amRange.location == NSNotFound && pmRange.location == NSNotFound);
+	
+	return is24h;
```

Comment:
Could please remove the `[self is24hTimeFormat]` check (and method) and set the locale explicitly to `en_GB`?